metacatalog/ext/standard_export/util.py

```diff
@@ -0,0 +1,627 @@
+                    for k, v in detail_dict['value'].items():
+                        expand = {
+                            f"{detail_dict['key']}.{k}": dict(
+                            value=v,
+                            key=detail_dict['key'],
+                            entry_uuid=detail_dict['entry_uuid'],
+                            description=detail_dict.get('description', 'nan')
+                            )
+                        }
+                        _details.update(expand)
+                # un-nested details
+                else:
+                    _details[detail_dict['key']] = detail_dict
+                    # remove unwanted key-value pairs
+                    _details[detail_dict['key']] = {key: val for key, val in _details[detail_dict['key']].items() if key in ['value', 'key', 'entry_uuid', 'description']}
+
+        # turn into a transposed dataframe
+        df = pd.DataFrame(_details).T
+
+        # append markdown table to details
+        details.append(df.to_markdown())
+
+    # if there are more than one entries in the ImmutableResultSet, a entry_uuid-indexed dictionary of details is returned
+    elif isinstance(rs.get('details'), dict):
+        for entry_uuid, entry_details_list in rs.get('details').items():
+            for detail_dict in entry_details_list:
+                # nested details
+                if isinstance(detail_dict['value'], dict):
+                    # include top-level detail of nested detail
+                    _details[detail_dict['key']] = detail_dict.copy()
+                    _details[detail_dict['key']]['value'] = 'nested'
+                    
+                    # remove unwanted key-value pairs
+                    _details[detail_dict['key']] = {key: val for key, val in _details[detail_dict['key']].items() if key in ['value', 'key', 'entry_uuid', 'description']}
+
+                    # go for nested details
+                    for k, v in detail_dict['value'].items():
+                        expand = {
+                            f"{detail_dict['key']}.{k}": dict(
+                            value=v,
+                            key=detail_dict['key'],
+                            entry_uuid=detail_dict['entry_uuid'],
+                            description=detail_dict.get('description', 'nan')
+                            )
+                        }
+                        _details.update(expand)
+                # un-nested details
+                else:
+                    _details[detail_dict['key']] = detail_dict
+                    # remove unwanted key-value pairs
+                    _details[detail_dict['key']] = {key: val for key, val in _details[detail_dict['key']].items() if key in ['value', 'key', 'entry_uuid', 'description']}
+
+            # turn into a transposed dataframe
+            df = pd.DataFrame(_details).T
+
+            # append markdown table to details
+            details.append(df.to_markdown())
```

Comment:
This is not ideal behavior. 
The details should end up as extra metadata info in the iso. It has an entity for that. Just putting tables into the abstract is not the final behavior.

Maybe we keep the function for now, but implement the proper behavior as soon as possible.
Thus, this should open an urgent issue.